gdc_client/parcel/segment.py

```diff
@@ -28,17 +30,6 @@
 )
 from gdc_client.parcel.const import SAVE_INTERVAL
 
-if OS_WINDOWS:
-    WINDOWS = True
-    from six.moves.queue import Queue
```

Comment:
To be safe, let's limit the changes in the PR to only the removal of `six`.    Let's just remove the `six` queue for the py3 `queue.Queue`.  Let's leave the windows specific code in place.